lib/TPP/Dialect/Transform/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 add_mlir_dialect_library(TPPLinalgXTransformOps
     LinalgXTransformOps.cpp
     IteratorCollapsing.cpp
+    SCFParallelLoopTiling.cpp
```

Comment:
Why is this in the transform dialect? We're not using the transform dialect here, just passes.